docs/index.md

```diff
@@ -0,0 +1,141 @@
+<!--meta-description
+WordPress.wasm is a client-side WordPress that runs without a PHP server thanks to the magic of WebAssembly.
+-->
+
+# Introduction
+
+- [What is WordPress.wasm?](#what-is-wordpress-wasm?)
+- [Getting started](#getting-started)
+- [Hello world](#hello-world)
+
+---
+
+### What is WordPress.wasm?
+
+WordPress.wasm is a client-side WordPress that runs without a PHP server thanks to the magic of WebAssembly.
+
+See the [live demo](https://wasm.wordpress.net/wordpress-browser.html).
+
+WordPress.wasm can power:
+
+* Runeditable code examples in your documentation or course
+* Plugin and theme demos in a private WordPress instance where the user is already logged in as admin
+* Easily switching PHP and WordPress version when testing
+* Replaying and fixing the failed CI tests right in the browser
+
+Intrigued? See [the full announcement post on make.wordpress.org](https://make.wordpress.org/core/2022/09/23/client-side-webassembly-wordpress-with-no-server/).
+
+WordPress.wasm provides a strong foundation for the above use-cases but does not implement them just yet. This project is still in its early days and needs contributors. Become a contributor today and help us make these tools a reality!
+
+---
+
+### Getting started
+
+The best way to play with WordPress.wasm is to clone and modify this repository. Seriously. There is no npm package or an easy-to-use public API just yet. We'll get there soon.
+
+Run the following commands to get started:
+
+```js
+git clone https://github.com/WordPress/wordpress-wasm
+cd wordpress-wasm
+npm install
+npm run dev
+```
+
+A browser should open and take you to your very own client-side WordPress at http://127.0.0.1:8777/wordpress-browser.html! 
+
+From here you can start playing with the code – [src/web/](https://github.com/WordPress/wordpress-wasm/tree/trunk/src) is a good place to start.
+
+## Repository structure
+
+WordPress.wasm consists of:
+
+* PHP to WASM compilation pipeline
+* WordPress bundler
+* A JavaScript library to run WordPress in the browser
+* A JavaScript library to run WordPress in Node.js
+
+### PHP to WASM compilation pipeline
+
+WordPress cannot run without PHP, so this repo embeds a set of scripts to compile PHP to a `.wasm` file. You'll find them in `wasm-build/php`. 
+
+#### Running the build scripts
+
+PHP builds can be conveniently triggered using npm scripts:
+
+* `npm run build:php:web` compiles PHP to a web-optimized `.wasm` binary
+* `npm run build:php:node` compiles PHP to a node.js-optimized `.wasm` binary
+
+The `.wasm` binaries are standalone. The web version doesn't require the node.js binary and vice versa.
+
+Here's what the build scripts do:
+
+1. Build the docker image
+2. Build the php.wasm binary
+3. Copy php.wasm to a target directory
```

Comment:
Here's a list of all build scripts, copied from [this PR comment](https://github.com/WordPress/wordpress-wasm/pull/25#issuecomment-1270356783) and updated to reflect the newest `package.json`. I figured it might be helpful to document them somewhere.

As a rough draft, I added task descriptions. The wording and format may need improvement.

- dev - Build for development, watch files, rebuild on change - Alias for `dev:web`
  - dev:web - Build for web target - Runs `dev:web:\*` in parallel
    - dev:web:app - Runs `build:web` with watch option
    - dev:web:serve - Runs live server
  - dev:node - Build for node target - Runs `./src/node/command.mjs`
- build - Build for production - Alias for `build:web`
  - build:web - Build web app, service worker, web worker, HTML, PHP in `dist-web` folder - Runs `build.js` for web target
  - build:node - Build Node.js app - Runs `build.js` for node target
  - build:php:web - Build PHP to WASM for web target - Runs `wasm-build/php/build-wasm.sh` for web target
  - build:php:node - Build PHP to WASM for node target - Runs `wasm-build/php/build-wasm.sh` for web target
  - build:wp - Create WordPress bundle `wp.data`
  - build:wpnet - Runs `build:web` with remote worker origin `wordpress.net`
  - build:app - Runs `build:web` and `build:node` sequentially
- clean - Remove built artifacts - Runs `clean:\*` in parallel
  - clean:php - Remove built artifacts in `dist-web/wasm-build/php`
  - clean:wp - Remove built artifacts in `dist-web/wasm-build/wordpress`